crates/board/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Patch
 
+- Update dependencies
```

Comment:
Just for example, here there is already a pre-release, so we just add the change to the appropriate sub-section (and bump the pre-release version in case we add a sub-section with the highest priority).